deploy/examples/monitoring/localrules.yaml

```diff
@@ -559,7 +559,7 @@ spec:
             description: >
               Data redundancy may be reduced, or is at risk, since one or more OSDs are at or above their
               'full' threshold. Add more capacity to the cluster, or delete unwanted data.
-        - alert: CephPGUnavilableBlockingIO
+        - alert: CephPGUnavailableBlockingIO
```

Comment:
@pcuzner Any concerns with renaming the alert? This would also be included in the ceph PR here: https://github.com/ceph/ceph/pull/46824

Reply:
Seems benign, so will ignore this question for merge.